contracts/SpokePool.sol

```diff
@@ -1641,24 +1641,7 @@ abstract contract SpokePool is
     }
 
     function _getV3RelayHash(V3RelayData memory relayData) private view returns (bytes32) {
```

Comment:
Should we consider making this public? That would permit people to throw a relayData object at this function and get a SpokePool-derived hash back. This could be used by integrators to verify their local hash derivation implementations.

Reply:
I don't see a downside either. Done here: https://github.com/across-protocol/contracts/pull/867/commits/fee01d917109ffa0d8a9120fddea6ff29bb4f115